code/__DEFINES/admin.dm

```diff
@@ -9,8 +9,10 @@
 // Skyrat Edit Addition - Mute LOOC
 #define MUTE_LOOC (1<<5)
 // Skyrat Edit End
+#define MUTE_INTERNET_REQUEST (1<<6)
```

Comment:
@Paxilmaniac Had to change this from the original TG pr because a skyrat edit it was originally 1<<5

Reply:
Fair enough I guess? Personal observation though, wouldn't it make more sense to keep MUTE_INTERNET_REQUEST as (1<<5) and then append our MUTE_LOOC as (1<<6) after that rather than having to shift the number of any and all potential future mutes

Reply:
ah true i can fix that